AGENTS.md

```
@@ -470,6 +623,45 @@ backend = Map.get(args, :backend)
 
 Only convert keys the schema declares, so caller input can never mint atoms.
 A string key arriving past that boundary is a programming error: raise.
+### 5.4b Normalize parameters — whitelist known keys, drop unknowns
```

markdownlint-cli2 (0.23.2)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Above

(MD022, blanks-around-headings)